### DISCUSSION TITLE
Incomplete characters in Brazilian Portuguese language

### DISCUSSION THREAD

**Discussion by ghost** [mriscoc/Ender3V2S1#292] [Q&A]
I have an Ender 3 V2 with DACAI screen and I have compiled the latest firmware for Brazilian Portuguese but I noticed that some characters are not displayed correctly and some words have not yet been translated.
I'm in the mood to help with the firmware translation and was wondering if this character issue is because of my screen version or is it some extra configuration I need to do?
In the file configuration.h I informed in pt_br language

![IMG_20220818_065902](https://user-images.githubusercontent.com/62189156/185382544-b64b0870-1e35-4034-b2c0-2a8d01ef87f1.jpg)

**Comment by mriscoc** [comment]
DWIN/DACAI screens have a limited charset. Could be interesting to add a language file where only simple chars are used.

**Reply by ghost** [reply]
> DWIN/DACAI screens have a limited charset. Could be interesting to add a language file where only simple chars are used.

So this DACAI screen is full of limitations because it doesn't display thumbnails either.
